packages/space-opera/src/components/mobile_view/utils.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {GltfModel} from '@google/model-viewer-editing-adapter/lib/main';
+
+export const DOMAIN = 'https://piping.glitch.me/';
+
+export function getRandomInt(max: number): number {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+// ex: 'https://piping.nwtgck.repl.co/123-456'
+export function getSessionUrl(
+    pipeId: number|string, sessionId: number): string {
+  return `${DOMAIN}${pipeId}-${sessionId}`;
+}
+
+// ex: 'https://piping.nwtgck.repl.co/ping-123'
+export function getPingUrl(pipeId: number|string) {
+  return `${DOMAIN}ping-${pipeId}`;
+}
+
+// ex: 'https://piping.nwtgck.repl.co/123-456-789'
+export function gltfToSession(
+    pipeId: number|string, sessionID: number, modelId: number): string {
+  return `${DOMAIN}${pipeId}-${sessionID}-${modelId}`;
+}
+
+// ex: 'https://piping.nwtgck.repl.co/123-456-env'
+export function envToSession(
+    pipeId: number|string, sessionID: number, envIsHdr: boolean): string {
+  const addOn = envIsHdr ? '#.hdr' : '';
+  return `${DOMAIN}${pipeId}-${sessionID}-env${addOn}`;
+}
+
+// ex: 'https://piping.nwtgck.repl.co/123-456-789'
+export function usdzToSession(
+    pipeId: number|string, sessionID: number, modelId: number): string {
+  return `${DOMAIN}${pipeId}-${sessionID}-${modelId}.usdz`;
```

Comment:
fixes iOS by adding usdz file type to url

Reply:
Have we checked this works with a .reality file too?